### PR TITLE
AC-976:Made the text in the text boxes displayed correctly in dark mode

### PR DESCRIPTION
**Pull request by Akshatji800**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-976

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/res/layout/fragment_add_provider.xml [hunk 1]
```diff
@@ -89,7 +89,7 @@
             android:hint="@string/identifier_hint"
             android:inputType="textCapWords"
             android:maxLines="1"
-            android:textColor="@color/dark_grey_8x"
+            android:textColor="@color/white"
```
Comment (HerbertYiga) [inline openmrs-client/src/main/res/layout/fragment_add_provider.xml:92]: cc @LuGO0

**Comment by shivamsawlani02** [comment]
@Akshatji800 please add screenshots.

**Comment by Akshatji800** [comment]
> @Akshatji800 please add screenshots.

AC-976

**Comment by shivamsawlani02** [comment]
@Akshatji800 can you add screenshots for light mode also?

**Comment by LuGO0** [comment]
Hi @Akshatji800 can you add screenshots since its a UI related change?

**Comment by Akshatji800** [comment]
> Hi @Akshatji800 can you add screenshots since its a UI related change?

dark-mode-final
light-mode-final

**Comment by Akshatji800** [comment]
@LuGO0 you can check now sorry for the late reaction I was facing some health issues.

**Comment by LuGO0** [comment]
No Issues take Care !! LGTM.

**Pull request merged**